src/page.ts

```diff
@@ -39,6 +43,11 @@ export type WaitForNetworkIdleOptions = {
   idleConnections?: number;
 };
 
+export type SandboxOptions = {
+  sandbox?: boolean | Network_ResourceType[];
+  sandboxPrompt?: boolean;
+};
+
```

Comment:
Could you add JSDocs explaining what these options do?

Reply:
I'll add the docs once it's decided what API should be adopted 👍